include/benchmark/benchmark.h

```diff
@@ -430,10 +430,50 @@ struct Statistics {
       : name_(name), compute_(compute) {}
 };
 
+// Converts whatever the stored manual time is to
+// seconds, which is used to estimate iterations
+// and similar
+typedef double(BenchmarkTimeCostFunc)(double);
+// default assumes measurements are done in seconds
+// (as they are currently done in the benchmark runner)
+inline double DefaultBenchmarkTimeCostFunc(double measurement) {
+  return measurement;
+}
+
 struct BenchmarkInstance;
 class ThreadTimer;
 class ThreadManager;
 
+class BenchmarkTime {
+ public:
+  BenchmarkTime();
+
+  void SetManualCostFunction(BenchmarkTimeCostFunc* to_time_cost_in_seconds);
```

Comment:
I'm not sure we want to spell `Manual` here.
I understand that now it only makes sense for manual time,
but if i do proceed into custom timers, it will be needed for non-manual time,
and i think we'll end up with either two identical functions with and without `Manual`,
or forever stuck with this function.

Reply:
I guess I'll remove the Manual then...!